utils.sh

```diff
@@ -238,7 +243,24 @@ function generate_templates {
     fi
 
     # clouds.yaml
-    go run metal3-templater.go clouds.yaml.template "$CLUSTER_PRO_IF" "$PROVISIONING_NETWORK" "$MACHINE_OS_IMAGE_URL" "$BOOTSTRAP_PROVISIONING_IP" "$CLUSTER_PROVISIONING_IP" > clouds.yaml
+    if [[ "$OPENSHIFT_VERSION" != "4.6" ]]; then
+      go run metal3-templater.go clouds.yaml.template "$CLUSTER_PRO_IF" "$PROVISIONING_NETWORK" "$MACHINE_OS_IMAGE_URL" "$BOOTSTRAP_PROVISIONING_IP" "$CLUSTER_PROVISIONING_IP" > clouds.yaml      # For compatibility with metal3-dev-env openstackclient.sh
+      # which mounts a config dir into the ironic-client container
+      mkdir -p _clouds_yaml
+      cp clouds.yaml _clouds_yaml
+    fi
+}
+
+function generate_auth_template {
+    MACHINE_OS_IMAGE_URL="http:///$(wrap_if_ipv6 $MIRROR_IP)/images/${MACHINE_OS_IMAGE_NAME}?sha256=${MACHINE_OS_BOOTSTRAP_IMAGE_SHA256}"
+    # Get the auth information for ironic and ironic-inspector
+    export KUBECONFIG="${SCRIPTDIR}/ocp/${CLUSTER_NAME}/auth/kubeconfig"
```

Comment:
This should be set up by whatever is calling this function; imho this is not something we should be exporting from inside random function calls.
If it's not set already at the time this is called then we should just pass it as a CLI argument rather than changing the environment.

Reply:
I'm going to test without setting the KUBECONFIG and see how it goes

Reply:
`common.sh` already sets `KUBECONFIG`.